### ISSUE TITLE
Shouldn't an "accession number" be unique?

### ISSUE THREAD

**Issue by agrogers** [Librarika/Issues#467]
Hi

I was working on the import of  a file and assumed that an "accession number" would be a unique number identifying an item. From WIKI:

"In libraries, art galleries, museums and archives, **an accession number is a unique identifier assigned to, and achieving initial control of, each acquisition**. Assignment of accession numbers typically occurs at the point of accessioning or cataloging."

However, even though my import provides an Accession Number, if i import  the same file a second time it creates a new title with the **same** Accession number. So it's no longer unique.

I think a smarter way to do the import is to check if the Accession Number already exists in the system. If it does exist then simply update the entry or ignore it altogether. 

Adding another item with the same accession number seems to be a bad option.

What do you think?
Andrew

**Comment by rayhan** [comment]
Yes, its true, importer doesn't not check the unique ness of accession number, it mainly depends on librarians data. We will look into this to further tune the process.

Thanks!

**Comment by agrogers** [comment]
Thanks.

**Comment by WebShapedBiz** [comment]
Same here.
